pages/desktop/base.py

```diff
@@ -247,19 +259,29 @@ def more_menu(self, item=None):
 
     def click_developer_hub(self):
         self.find_element(*self._devhub_locator).click()
-        self.wait.until(EC.number_of_windows_to_be(2))
+        self.wait.until(
+            EC.number_of_windows_to_be(2),
+            message=f'Number of windows was {len(self.selenium.window_handles)}',
+        )
         new_tab = self.selenium.window_handles[1]
         self.selenium.switch_to_window(new_tab)
         self.wait.until(
-            EC.visibility_of_element_located((By.CLASS_NAME, 'DevHub-Navigation-Logo'))
+            EC.visibility_of_element_located((By.CLASS_NAME, 'DevHub-Navigation-Logo')),
+            message=f'DevHub homepage not loaded; page was {self.selenium.current_url}',
         )
 
     def click_extension_workshop(self):
         self.find_element(*self._extension_workshop_locator).click()
-        self.wait.until(EC.number_of_windows_to_be(2))
+        self.wait.until(
+            EC.number_of_windows_to_be(2),
+            message=f'Number of windows was {len(self.selenium.window_handles)}',
```

Comment:
could we add an "expected 2" at the end of this?
```suggestion
            message=f'Number of windows was {len(self.selenium.window_handles)}, expected 2',
```